### DISCUSSION TITLE
Arduino reset

### DISCUSSION THREAD

**Discussion by ginrider1** [Stellarium/stellarium#3234] [Q&A]
Hello!
I've faced the situation that when telescope control is activated the Stellarium changes DTR serial port signal which causes MCU to reset. Can someone help to solve this? Thanks!

**Comment by 10110111** [comment]
DTR by itself cannot lead to MCU reset. It's must be something in your firmware that does.

**Comment by ginrider1** [comment]
There is a capacitor between FT's DTR and MCU's reset pin. Provides reset signal for programming via USB UART.
